packages/create-cloudflare/templates-experimental/remix/templates/wrangler.json

```diff
@@ -1,11 +0,0 @@
-{
-  "name": "<TBD>",
-  "main": "./server.ts",
-  "compatibility_date": "<TBD>",
-  "assets": {
-    "directory": "./build/client"
-  },
-  "observability": {
-    "enabled": true
-  }
-}
```

Comment:
Will this will removes the ability to update the name and compat date? FYI, the wrangler config from the remix template use custom build which works slightly different then what we are suggesting in c3. 🤔

Reply:
Oh interesting, is it not valid for C3? In that case I'll change this to TOML so it keeps overwriting the remix one, but we should probably fix that upstream?

Reply:
Added back in https://github.com/cloudflare/workers-sdk/pull/7676/commits/f1d26e526da987489ab03af747c5f2dc029523f4